src/nevermined/utils/WebServiceConnector.ts

```diff
@@ -123,6 +123,37 @@ export class WebServiceConnector extends Instantiable {
         return d
     }
 
+    public async getFile(
+        url: string,
+        index?: number,
+        headers?: any,
+    ): Promise<{
+        file: Response;
+        filename: string;
+    }> {
+        const response = await this.get(url, headers)
+        if (!response.ok) {
+            throw new Error('Response error.')
+        }
+        let filename: string
+        try {
+            [, filename] = response.headers
+                .get('content-disposition')
+                .match(/attachment;filename=(.+)/)
+        } catch {
+            try {
+                filename = url.split('/').pop()
+            } catch {
+                filename = `file${index}`
+            }
+        }
+
+        return {
+            file: response,
```

Comment:
the response is not a file. We should return a file instance `File` instance that already contains a `name` property